marklogic-data-hub/src/test/java/com/marklogic/hub/EndToEndFlowTests.java

```diff
@@ -799,7 +857,7 @@ else if (prefix.equals("1x-legacy")) {
             if (dataFormat.equals(DataFormat.JSON)) {
                 String expected = getResource("e2e-test/" + filename + "." + dataFormat.toString());
                 String actual = stagingDocMgr.read("/input" + fileSuffix + "." + dataFormat.toString()).next().getContent(new StringHandle()).get();
-                assertJsonEqual(expected, actual, false);
+                 assertJsonEqual(expected, actual, false);
```

Comment:
looks like an accidental space here

Reply:
are you talking about my brain? lol. yes. this is.